sink/stdout.go

```diff
@@ -51,7 +51,7 @@ func (s *StdoutSink) Stop() {
 }
 
 // Put ..
-func (s *StdoutSink) Put(data []byte) error {
-	fmt.Println(string(data))
+func (s *StdoutSink) Put(key string, data []byte) error {
+	fmt.Printf("key: %s, message: %s\n", key, string(data))
```

Comment:
I think this could break anything that expects to consume the stdout sink as JSON-encoded log lines.

Reply:
ok will remove, though it's helpful to print the key for testing purposes